gradlew

```diff
@@ -0,0 +1,244 @@
+#!/bin/sh
```

Comment:
Does this file need to be in the root? Would be nice to have it live in the `dev/kotlin` folder just for the sake of uncluttering

Reply:
Unfortunately yes. I tried moving it into the kotlin folder in these commits: [f16eced](https://github.com/xmtp/proto/pull/31/commits/f16eced15ef39a887c223b0190b28a3b42dddfdd) [9b8fbe2](https://github.com/xmtp/proto/pull/31/commits/9b8fbe2497aa8352d7b837b58aa8fc960870a903) but ran into errors with [gradle wrapper validation](https://github.com/gradle/wrapper-validation-action) which looks for the wrapper in the root.

Reply:
OK. Not the end of the world

Reply:
If we do move forward with a java package as well of the protos this will be reused between kotlin and java.